mesos-docker/run/mit-application.conf.template

```diff
@@ -0,0 +1,18 @@
+#Provide full path to the mesos native library. For ubuntu its filename is libmesos.so, for mac its libmesos.dylib
+mit.mesos.native.library.location = "replace_with_mesos_lib"
+
+#Used to copy the application jar file for cluster mode tests
+mit.hdfs.uri = "replace_with_hdfs_uri"
+
+#The IP address of the host as seen from docker instance. This ip address is used to connect to runner
+#from docker instance.
+mit.docker.host.ip = "replace_with_docker_host_ip" 
+
+#path of mesos slaves to find spark executor. This should be set when you create docker based mesos cluster
+mit.spark.executor.tgz.location = "replace_with_executor_tgz_location"
```

Comment:
Is this the same as `spark.executor.uri`? Let's use that, it's probably clearer what it means.